ios/brave-ios/Sources/Brave/Frontend/Rewards/BraveRewards.swift

```diff
@@ -219,14 +219,12 @@ public class BraveRewards: NSObject {
     if let innerText = adsInnerText, ads.isServiceRunning() {
       ads.notifyTabHtmlContentDidChange(
         tabId,
-        url: url,
-        redirectChain: redirectionURLs ?? [],
+        redirectChain: redirectionURLs ?? [url],
```

Comment:
For consistency, maybe rename `redirectionURLS` to `redirectChain`

Reply:
Fixed